xgboost_ames_housing/README.md

```diff
@@ -35,7 +35,7 @@ directly download the image.
 
 ```
 IMAGE_NAME=ames-housing
-VERSION=v1
+VERSION=latest
```

Comment:
Probably better to pin a specific version instead of using latest.

Reply:
In case of codelab/examples instructions, it is always better to use latest tag instead of a specific tag, because readers should not care about the version of image.  You can see that [hello_world example](https://docs.docker.com/get-started/) for docker.

Internally, we can add various version tags to the images if we want to track the image changes, as one image can have multiple tags.